### ISSUE TITLE
Use temporary file for downloads?

### ISSUE THREAD

**Issue by rod-glover** [pacificclimate/dash-dv-explorer#45]
At present, the files created for downloading data persist forever. This is probably not desirable long-term, although we can just delete files older than a given age to these files out. They are small.

**Comment by jameshiebert** [comment]
Yes, we should use temporary files. The files may be small, but keep in mind that this will be running in a tiny container and should have minimal storage attached to it. Also, the service should run forever, so we don't want to have to shell into the container to remove things... that'd be a pain. It should clean up on its own.
